### ISSUE TITLE
Document how to use @allowextraarguments

### ISSUE THREAD

**Issue by davidavdav** [zachmatson/ArgMacros.jl#9]
That is, how do I access these leftover arguments?  `ARGS` still gives me all the arguments, including the parsed arguments. 

Or is there another way to have multiple arguments end up in a `Array{String}`?

Use case: filenames than the script needs to process.

Thanks!

**Comment by zachmatson** [comment]
Currently, `@allowextraarguments` just tells ArgMacros not to exit your script when it notices extra arguments. I’m happy to add a feature to address the use case you mention though.

Would a `@positionalleftover varname` macro or similar to capture an `Array{String}` of additional arguments meet your needs here?

**Comment by davidavdav** [comment]
Hello, 

Thanks for the quick reply.  Yes, I believe that would be very useful.  Especially because then I don't have to wrap the script in a bash `for file in $files; do julia-script $file; done` which has some overhead in startup, compiling and parsing. 

BTW your package is extremely useful!  I still use "small script pipeline programming" at the daily level, and ArgMacros finally makes Julia compete with Python again (also the improved start-up time helps I suppose).  Now just an easy way to cache compiled scripts...

I recently needed to compare very large word strings, which is a N^2 problem I figure.  I happened to have lying around a [Julia package](https://github.com/davidavdav/WordErrorRate.jl) that I only needed to update to somewhat more modern Julia, and it really is way faster that its Python transliteration.
